doc/getting-started/apl-nuc.rst

```diff
@@ -68,29 +68,206 @@ complete this setup.
    Please refer to the ACRN :ref:`release_notes` for the Clear Linux OS
    version number tested with a specific ACRN release.  Adjust the
    instruction below to reference the appropriate version number of Clear
-   Linux OS (we use version 27960 as an example).
+   Linux OS (we use version 28960 as an example).
 
 #. Download the compressed Clear Linux OS installer image from
-   https://download.clearlinux.org/releases/27960/clear/clear-27960-installer.img.xz
+   https://download.clearlinux.org/releases/28960/clear/clear-28960-installer.img.xz
    and follow the `Clear Linux OS installation guide
    <https://clearlinux.org/documentation/clear-linux/get-started/bare-metal-install>`__
-   as a starting point for installing Clear Linux OS onto your platform.  Follow the recommended
-   options for choosing an **Manual (Advanced)** installation type, and using the platform's
+   as a starting point for installing Clear Linux OS onto your platform. Follow the recommended
+   options for choosing an **Advanced options** installation type, and using the platform's
    storage as the target device for installation (overwriting the existing data
    and creating three partitions on the platform's storage drive).
 
    High-level steps should be:
 
-   #.  Install Clear Linux OS on a NUC using the "Manual (Advanced)" option.
-   #.  Use default partition scheme for storage
-   #.  Name the host "clr-sos-guest"
-   #.  Add an administrative user "clear" with "sudoers" privilege
-   #.  Add these additional bundles "editors", "user-basic", "desktop-autostart", "network-basic"
-   #.  For network, choose "DHCP"
+   #.  Launch the Clear Linux OS installer boot menu
+   #.  With Clear Linux OS highlighted, select Enter.
+   #.  From the Main Menu, select "Configure Media" and set "Auto Partition" to your desired hard disk.
+   #.  ``shift + A`` to the "Advanced options".
+   #.  Select "Additional Bundle Selection" to add additional bundles "desktop-autostart", "editors", "network-basic", "user-basic"
+   #.  Select "User Manager" to add an administrative user "clear"
```

Comment:
The current Clear Linux installer requires you to set the password for the user now (not when you first log in).  So change to:
```
#.  Select "User Manager" to add an administrative user "clear", and assign a password.
```